vendor/modules.txt

```diff
@@ -312,6 +312,9 @@ go.uber.org/goleak/internal/stack
 golang.org/x/crypto/md4
 golang.org/x/crypto/ocsp
 golang.org/x/crypto/ripemd160
+# golang.org/x/crypto/x509roots/fallback v0.0.0-20231030152948-74c2ba9521f1
+## explicit; go 1.20
+golang.org/x/crypto/x509roots/fallback
```

Comment:
This is because the SetFallbackRoots was added in 1.20.

I might be able to work around this if building with < 1.20 is required.

Reply:
There is no need for a workaround :+1: So far we do support the two last golang's versions, which aligns the go team